api/libretimeapi/tests/models/test_schedule.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime, timedelta
+
+from django.test import SimpleTestCase
+from libretimeapi.models import Schedule, Show, ShowInstance
+
+
+class TestSchedule(SimpleTestCase):
+    def test_get_cueout(self):
```

Comment:
Nit - you could break these up into different functions with a class method to preconfigure. I'm happy either way

Reply:
Did a small refactoring, but I won't do more magic for today. It can be refactored properly into class methods once a second test needs some of the data.